proposals/913-components-repo/README.md

```diff
@@ -0,0 +1,461 @@
+# KEP-913: Dedicated Repository for Kubeflow Pipelines Components & Pipelines
+
+<!-- toc -->
+
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [Repository Layout](#repository-layout)
+    - [Core (components and pipelines)](#core-components-and-pipelines)
+    - [Third-Party (third_party)](#third-party-third_party)
+  - [Artifact Metadata Schema](#artifact-metadata-schema)
+  - [Standardized README Templates](#standardized-readme-templates)
+  - [Linting & Continuous Integration](#linting--continuous-integration)
+  - [Pytest Structure and Local Execution](#pytest-structure-and-local-execution)
+  - [Maintenance Automation](#maintenance-automation)
+  - [Onboarding & Documentation](#onboarding--documentation)
+  - [Packaging & Release Management](#packaging--release-management)
+  - [Governance & Tiers](#governance--tiers)
+  - [Open Questions](#open-questions)
+- [Design Details](#design-details)
+  - [Implementation Phases](#implementation-phases)
+  - [Rollout and Migration](#rollout-and-migration)
+- [Risks and Mitigations](#risks-and-mitigations)
+- [Test Plan](#test-plan)
+  - [Unit & Compile Checks](#unit--compile-checks)
+  - [Continuous Integration](#continuous-integration)
+- [Graduation Criteria](#graduation-criteria)
+- [Implementation History](#implementation-history)
+- [Drawbacks](#drawbacks)
+- [Alternatives](#alternatives)
+  - [Separate Repositories](#separate-repositories)
+  - [Keep the Existing `components` Directory in the KFP Repo](#keep-the-existing-components-directory-in-the-kfp-repo)
+  <!-- /toc -->
+
+## Summary
+
+Establish a dedicated Kubeflow Pipelines (KFP) repository\* that hosts reusable components and full pipelines under a
```

Comment:
We have a robust internal product offering that implements much of what this document describes. I will encourage the internal maintainers to explore the possibility of contributing.